ASWF/meetings/tsc/2020-10-05.md

```diff
@@ -0,0 +1,128 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+October 5, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [ ] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [X] Mark Titchener (_TSC_) - Foundry
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [ ] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Thomas Mansencal - Weta Digital
+  * [X] Aymee Yeh - Netflix
+  * [X] J Schulte - Industrial Light & Magic
+  * [X] Matthias Scharfenberg - Industrial Light & Magic
+
+Apologies
+  * Doug Walker
+
+# **OCIO TSC Meeting Notes**
+
+* Review pending PRs:
+    - #1160 - C++ API docs:
+        - Initial work and structure of C++ API docs. Feedback on organization 
+          of content welcome. Can look at RST or build locally. Also includes 
+          some general CMake changes and removal of auto-install of 
+          documentation-specific dependencies.
+        - Patrick: Noticed this morning on tsc-private topic related to 
+          copyright of images found with license scan. Needs a look.
+    - #1159 - Improve CMake files:
+        - Patrick: Not C++. Faced issues with CMake on Windows and tried to 
+          fix.
+    - #1157 - FileTransform interpolation:
+        - Patrick: Last meeting discussed v1 not throwing if interpolation was 
+          missing. Bernard checked and it is throwing. Explanation in pull 
```

Comment:
I mentioned that `Bernard checked and it is throwing for FileTransform (i.e. LUTs) as expected.`

Reply:
Updated.